backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -510,6 +510,37 @@ def __str__(self):
         return f"Case review for {self.destruction_list} ({self.destruction_list_item})"
 
 
+class DestructionListCoReview(models.Model):
+    """
+    Model linking a co reviewer (User) to a destruction list.
+    Actual (temporary) review items are stored using selection API
+    """
+
+    destruction_list = models.ForeignKey(
+        DestructionList,
+        on_delete=models.CASCADE,
+        related_name="co_reviews",
+        verbose_name=_("destruction list"),
+    )
```

Comment:
At this moment we don't have any information to which "round of review" this refers. Could this be a problem for when a main reviewer needs to re-review?

Reply:
I thought about that but it's unclear whether the review needs to be "reset". My assumption is that when the list gets back after processing it's way smaller and doesn't need a new co-review (but I may be wrong).

Do you have any suggestions here?

Reply:
I am not sure, since at this point the "review" doesn't exist yet :thinking: The co-reviews also don't really contain any information about the individual cases, only a generic comment. And we only show the indication that a reviewer is done (the comment is only shown in the history tab).

Maybe the list co-reviews endpoint should have a way to filter co-reviews created "after the latest review"? then we could differentiate the rounds and show the "done" indication correctly :thinking:

Reply:
I've added a commit that only shows statuses of co reviews that are created after the latest review response (if any).